### PR TITLE
feat: Create the Service Interface and its implementation about the F…

### PR DESCRIPTION
**Pull request by BesmirKrasniqi**
…eedback entity

### CODE REVIEW

**Review by ArianitSGashi — APPROVED** [review]

File: src/OnlineVotingS.Application/Services/IService/IFeedbackService.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using OnlineVotingS.Application.DTO.PostDTO;
+using OnlineVotingS.Application.DTO.PutDTO;
+using OnlineVotingS.Domain.Entities;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace OnlineVotingS.Application.Services.IService
+{
```
Comment (AdhurimUP) [inline src/OnlineVotingS.Application/Services/IService/IFeedbackService.cs:11]: do not use '{' on the namespace, beautify with ';'

File: src/OnlineVotingS.Application/Services/ImplService/FeedbackService.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+using OnlineVotingS.Application.DTO.PutDTO;
+using OnlineVotingS.Application.Services.IService;
+using OnlineVotingS.Domain.Entities;
+using OnlineVotingS.Domain.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace OnlineVotingS.Application.Services.ImplService
+{
```
Comment (AdhurimUP) [inline src/OnlineVotingS.Application/Services/ImplService/FeedbackService.cs:14]: do not use '{' on the namespace, beautify with ';'

File: src/OnlineVotingS.Application/Services/ImplService/FeedbackService.cs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+        public async Task<Feedback> UpdateFeedbackAsync(FeedbackPutDTO feedbackDto)
+        {
+            var feedback = await _feedbackRepository.GetByIdAsync(feedbackDto.FeedbackID);
+            if (feedback == null)
+            {
+                throw new KeyNotFoundException($"Feedback with ID {feedbackDto.FeedbackID} not found.");
+            }
+
+            _mapper.Map(feedbackDto, feedback);
```
Comment (AdhurimUP) [inline src/OnlineVotingS.Application/Services/ImplService/FeedbackService.cs]: in the future we need to see if the mapping side should be in separated file!

**Review by AdhurimUP — CHANGES_REQUESTED** [review]
resolve comments!

File: src/OnlineVotingS.Application/Services/ImplService/FeedbackService.cs [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+using OnlineVotingS.Application.Services.IService;
+using OnlineVotingS.Domain.Entities;
+using OnlineVotingS.Domain.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace OnlineVotingS.Application.Services.ImplService
+{
+    public class FeedbackService : IFeedbackService
```
Comment (AdhurimUP) [inline src/OnlineVotingS.Application/Services/ImplService/FeedbackService.cs]: reformat the code in align, remove the typo empty spaces

**Review by AdhurimUP — APPROVED** [review]

**Pull request merged**